buổi-10/main.js

```diff
@@ -0,0 +1,150 @@
+// bài 1
+const getMultiplication = number => {
+    if (number >= 1 && number <= 10) {
+        for(let i = 1; i <= 10; i++) {
+            console.log(number + ' * ' + i + ' = ' + (number * i))
+        } 
+    } else {
+        console.log('Chỉ nhận giá trị từ 1-> 10!!')
+    }
+}
+
+// bài 2
+const factorial = number => {
+    if(number === 0){
```

Comment:
```suggestion
    if (number === 0) {
```
Xem kỹ cách đặt dấu cách